example-application.properties

```diff
@@ -10,3 +10,4 @@ datadirectory=
 
 # Ensure OpenAPI docs display correct URL when server is behind a proxy
 server.use-forwarded-headers=true
+server.forward-headers-strategy=framework
```

Comment:
Would you ever not want these options configured? If not, they should go in the internal `src/main/resources/application.yml` instead of instance-specific config

Reply:
Thanks! I've made this update.